packages/common/src/index.ts

```diff
@@ -370,6 +370,30 @@ export default class Common extends EventEmitter {
     return this.paramByHardfork(topic, name, hardfork)
   }
 
+  /**
+   * Checks if an EIP is activated by either being included in the EIPs
+   * manually passed in with the `eips` constructor option or in a
+   * hardfork currently being active
+   *
+   * Note: this method only works for EIPs being supported
+   * by the `eips` constructor option
+   * @param eip
+   */
+  isActivatedEIP(eip: number): boolean {
+    if (this.eips().includes(eip)) {
+      return true
+    }
+    for (const hfChanges of HARDFORK_CHANGES) {
+      const hf = hfChanges[1]
```

Comment:
Had to do a double take here, this is a bit hard to read (but is due to how these hardforks are exposed). But yeah, this looks fine.